packages/instantsearch.js/src/widgets/rating-menu/rating-menu.tsx

```diff
@@ -168,7 +168,7 @@ const renderer =
         templateProps={renderState.templateProps!}
         toggleRefinement={refine}
       >
-        <svg style="display:none;">
+        <svg xmlns="http://www.w3.org/2000/svg" style="display:none;">
```

Comment:
is this actually needed?

Reply:
I'd say no except for maybe really old browser versions, but I prefered to harmonize changes by adding instead of removing when possible.

Reply:
it's not required since html5, so I'm happy to leave them out everywhere